chain/src/main/java/org/ethereum/beacon/chain/observer/ObservableStateProcessorImpl.java

```diff
@@ -53,32 +52,35 @@
   private final Publisher<BeaconTuple> beaconPublisher;
 
   private static final int UPDATE_MILLIS = 500;
-  private ScheduledExecutorService regularJobExecutor;
-  private ScheduledExecutorService continuousJobExecutor;
+  private Scheduler regularJobExecutor =
```

Comment:
is it lazy? we don't need both until we need this service started

Reply:
Nope :/ 
Moved them back to `start()`